cabal/ganeti.template.cabal

```diff
@@ -80,7 +80,7 @@ library
 
   if flag(htest)
     build-depends:
-        HUnit                         >= 1.2.4.2    && < 1.3
+        HUnit                         >= 1.2.4.2    && < 1.7
```

Comment:
Change max HUnit version?

Reply:
Yes, that's a typo and it was so in the original commit as well. I'll fix the description, thanks!